packages/block-editor/src/components/block-list/index.js

```diff
@@ -196,7 +196,6 @@ class BlockList extends Component {
 			blockClientIds,
 			rootClientId,
 			__experimentalMoverDirection: moverDirection = 'vertical',
-			isDraggable,
```

Comment:
This prop was available so [alternative editor implementations could hide the drag icon](https://github.com/WordPress/gutenberg/pull/9569#discussion_r217317091) if they needed (mobile?). Perhaps it's fine to remove, but wanted to flag in case it's not.

Reply:
How successful was that? Any editor you know ever used it?